W-12006339-v25-dwfunction-chk-v3-duke (#236)
mulesoft/docs-dataweave: modules/ROOT/pages/dw-core-functions-typeof.adoc

=== Example

> This example shows that the type of a value is independent of the type it's

Author: ```suggestion
This example shows that the type of a value is independent of the type its
```

Author: added to source

> This example shows that the type of a value is independent of the type it's

Author: ```suggestion
This example shows that the type of a value is independent of the type with which it is declared.
```

> declared with.

Author: ```suggestion
```

> declared with.

Author: ```suggestion
```
